subprojects/docs/src/snippets/codeQuality/kotlin/build.gradle.kts

```diff
@@ -39,3 +39,12 @@ tasks.withType<Checkstyle>().configureEach {
     }
 }
 // end::customize-checkstyle-report[]
+
+// tag::customize-pmd[]
+pmd {
+    isConsoleOutput = true
+    toolVersion = "6.21.0"
+    rulePriority = 5
+    ruleSets("category/java/errorprone.xml", "category/java/bestpractices.xml")
```

Comment:
This is different from the Groovy version.  If you're trying to set the rule sets to a particular set, you need to use equals in both places, if you're just trying to add rule sets, I think you can use the same syntax for both.

Reply:
Great catch, fixed